buzz.html.slim

```diff
@@ -13,7 +13,7 @@ description: JBoss Developer is an open and active community, with updates happe
   .large-24.columns.buzz-filters
     form.buzz-filters
       input(type="text" name="buzz-filter-text" placeholder="Filter by Keyword")
-      .searchResults 
+    /  .searchResults 
```

Comment:
Yeah, you're probably right. I commented it out, looked good, committed :)

On Tue, Jan 27, 2015 at 6:05 AM, Paul Robinson <notifications@github.com>
wrote:

> In buzz.html.slim
> <https://github.com/jboss-developer/www.jboss.org/pull/752#discussion_r23605861>
> :
>
> > @@ -13,7 +13,7 @@ description: JBoss Developer is an open and active community, with updates happe
> >    .large-24.columns.buzz-filters
> >      form.buzz-filters
> >        input(type="text" name="buzz-filter-text" placeholder="Filter by Keyword")
> > -      .searchResults
> > +    /  .searchResults
>
> I don't think we need this as a comment
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-developer/www.jboss.org/pull/752/files#r23605861>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp